lib/main.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:math';
+
+import 'package:dart_collections_journey/names.dart';
+import 'package:word_generator/word_generator.dart';
+
+void main() {
+  separator(1);
+  task01();
+  separator(2);
+  task02();
+  separator(3);
+  task03();
+}
+
+void separator(int task) {
+  print('============ Task $task ============');
+}
+
+// Task 1: Списки (List)
+// Створіть список numbers зі 100 елементів. Кожен елемент — випадкове число від 0 до 100 (використайте Random().nextInt(101)).
+// Виведіть цей список на екран.
+// Виведіть на екран 65-й елемент списку у форматі: "65-й елемент: [значення]".
+// Вставте число 1000000000 на 50-ту позицію списку.
+// Видаліть зі списку елементи зі значеннями: 24, 45, 66, 88.
+// За допомогою циклу for переберіть список. Під час перебору:
+// Порахуйте суму всіх елементів, що діляться на 3 без залишку.
+// Виведіть результат.
+// Створіть порожній список temp. За допомогою циклу for-in переберіть numbers. Додайте до temp лише ті елементи, що діляться на 2 без залишку.
+// Виведіть довжину списку temp.
+void task01() {
+  List<int> numbers = List.generate(100, (i) => Random().nextInt(101));
+  print('список випадкових чисел: ${numbers.join(', ')}');
+  print('65-й елемент: ${numbers[64]}');
+  numbers.insert(49, 1000000000);
+  List<int> bannedNumbers = [24, 45, 66, 88];
+  numbers.removeWhere((e) => bannedNumbers.contains(e));
```

Comment:
You might also consider using a tear-off here for conciseness:
https://www.youtube.com/watch?v=OmCaloD7sis 
```suggestion
  numbers.removeWhere(bannedNumbers.contains);
```